workspace.json

```diff
@@ -0,0 +1,8 @@
+{
+  "version": 2,
+  "projects": {
+    "backend": "apps/backend",
+    "landing-page": "apps/landing-page",
+    "ui-components": "libs/ui-components"
```

Comment:
auxo app seems to miss here

Reply:
I intentionally left Auxo out as we've discussed this a few days ago that only the new next.js project will be included in nx.